samples/switchless/README.md

```diff
@@ -34,9 +35,9 @@ the associated costs and savings. In general, the good candidates for switchless
 1) short, thus the transition takes relatively high percentage of the overall execution time of the call; and
 2) called frequently, so the savings in transition time add up.
 
-## How does Open Enclave support switchless OCALLs
+## How does Open Enclave support switchless calls
 
-Open Enclave only supports synchronous switchless OCALLs currently. When the caller within an enclave makes a
+Open Enclave supports both synchronous switchless OCALLs and ECALLs currently. When the caller within an enclave makes a
```

Comment:
grammar: remove "both".

Also I don't think the sentence is true as worded, since I believe switchless calls are currently only supported for SGX.   Furthermore, I think the design inherently only supports TEEs that permit shared memory.

Reply:
I have reworded the sentence. Let me know if it sounds OK.